### ISSUE TITLE
Question for task 8

### ISSUE THREAD

**Issue by robert1003** [amjltc295/PythonHomework#74]
I found out that I need to put arial.ttf font file myself under folder /src to run successfully.
Is it normal?

P.S. I attached the font file I used.
[arial.ttf.zip](https://github.com/amjltc295/PythonHomework/files/2618019/arial.ttf.zip)

**Comment by amjltc295** [comment]
No, that probably means we need to fix some bugs in `utils.py`, as suggested in  the code in #66

**Comment by robert1003** [comment]
OK, thanks

**Comment by amjltc295** [comment]
By the way, are you using conda environment?

**Comment by robert1003** [comment]
Yes, This is my Python version:
Python 3.6.4 :: Anaconda custom (64-bit)

**Comment by amjltc295** [comment]
@robert1003 could you verify this bug and send a PR to fix it? It worths up to 30 bonus points.

**Comment by robert1003** [comment]
OK

**Comment by robert1003** [comment]
I updated my pull request to [Fix task 8 bug] with a arial.ttf file

**Comment by amjltc295** [comment]
@robert1003  Adding the `arial.ttf` it's not a good solution. We should consider changing the `utils.py` to use a default font. Also, your PR to fix the bug should not include your homework.

**Comment by robert1003** [comment]
OK, I'll fix it.

**Comment by robert1003** [comment]
I sent a new pr with the title [Fix task 8 bug]

**Comment by s950449** [comment]
I also make a PR

**Comment by amjltc295** [comment]
As someone reported that #90 may not work on MacOS,  I also fixed it in aa60ccd8123e27fa410adba8c10f29c5955da974 for local tests.
